base/src/main/java/io/spine/validate/AbstractValidatingBuilder.java

```diff
@@ -42,7 +46,7 @@
  * Serves as an abstract base for all {@linkplain ValidatingBuilder validating builders}.
  */
 public abstract class AbstractValidatingBuilder<T extends Message, B extends Message.Builder>
-        implements ValidatingBuilder<T, B> {
+        implements ValidatingBuilder<T, B>, Logging {
```

Comment:
Let's replace with a static method call.

A validating builder is a public Java API, which hardly needs to expose its `log` to the users.